Microsoft.Toolkit.Uwp.SampleApp/SamplePages/SlidableListItem/SlidableListItemPage.xaml

```diff
@@ -27,7 +28,8 @@
                                        IsRightSwipeEnabled="{Binding DataContext.IsRightSwipeEnabled.Value, ElementName=Root, Mode=OneWay}"
                                        IsLeftSwipeEnabled="{Binding DataContext.IsLeftSwipeEnabled.Value, ElementName=Root, Mode=OneWay}"
                                        LeftCommand="{x:Bind ToggleFavorite}"
-                                       RightCommandRequested="SlidableListItem_RightCommandActivated"
+                                       RightCommand="{Binding DeleteItem, ElementName=Page, Mode=OneWay}"
```

Comment:
Ugh, sorry, that's what happens when context gets removed in these code reviews, you are correct. Ignore this then.